addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -13328,4 +13328,40 @@ QUnit.module("Views", (hooks) => {
             );
         }
     );
+
+    QUnit.test("Kanban: hide no content helper on resequence", async (assert) => {
+
+        serverData.models.partner.records = []
+        await makeView({
+            type: "kanban",
+            resModel: "partner",
+            serverData,
+            arch:
+            `
+            <kanban sample="1">
+                <templates>
+                    <div t-name="kanban-box">
+                        <field name="display_name"/>
+                    </div>
+                </templates>
+            </kanban>`,
+            groupBy: ["product_id"],
+            async mockRPC(route, args) {
+                if (args.method === "web_read_group") {
+                    return {
+                        groups: [
+                            { __domain: [["product_id", "=", 3]], product_id_count: 0, product_id: [3, "hello"],},
+                            { __domain: [["product_id", "=", 5]], product_id_count: 0, product_id: [5, "xmo"], },
+                        ],
+                        length: 2,
+                    };
+                }
+            },
+        });
+        assert.containsOnce(target, ".o_view_nocontent");
+        const elements = document.querySelectorAll(".o_column_title");
+        await drag(elements[0], elements[1]);
+        await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
Why is it needed? 🧐

Reply:
Because It is failing test case if we don't do this
Also this drag should continue dragging and test case should pass but It doesn't working in assert :thinking: see runbot red build
P.S. Other drag related test cases are also failing in local but seems to be working on Runbot